templates/header.html

```diff
@@ -61,5 +63,13 @@
             signOut();
         });
     }
+
+    const maintabsEl = document.querySelector('#maintabs');
+    for (let tabEl of maintabs.children) {
```

Comment:
Since it is going through all children, does it mean             
(<li><a href="/metrics/css/popularity">CSS</a></li> or  <li><a href="/metrics/feature/popularity">JS/HTML</a></li>)
 and 
<div class="nav-dropdown-container flex-item" href="/metrics">

will all be `active`? when the page is on

Reply:
maintabs.children is a list of only the direct children, so only the <div> for the menu header gets the underline.